#! /bin/sh /usr/share/dpatch/dpatch-run
## 99_rename_mariadb-server_files_in <otto@seravo.fi>
##

@DPATCH@
=== modified file 'CMakeLists.txt'
--- old/CMakeLists.txt	2014-01-29 20:55:56.499842093 +0200
+++ new/CMakeLists.txt	2014-01-31 17:12:27.633594128 +0200
@@ -433,8 +433,8 @@
 
 IF(DEB)
   CONFIGURE_FILE(
-    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.files.in
-    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.files)
+    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.install.in
+    ${CMAKE_SOURCE_DIR}/debian/mariadb-server-5.5.install)
 ENDIF(DEB)
 
 # Handle the "INFO_*" files.
